mern-ecommerce/routes/user.js

```diff
@@ -18,7 +18,7 @@ router.get('/secret/:userId', requireSignin, isAuth, isAdmin, (req, res) => {
 
 router.get('/user/:userId', requireSignin, isAuth, read);
 router.put('/user/:userId', requireSignin, isAuth, update);
-router.get('/orders/by/user/:userId', requireSignin, isAuth, purchaseHistory);
+router.get('/orders/by/user/:userId', requireSignin, isAuth, purchaseHistoryy);
```

Comment:
Typo in `purchaseHistoryy`. It should be `purchaseHistory`. The correct line should be:
```
router.get('/orders/by/user/:userId', requireSignin, isAuth, purchaseHistory);
```